apps/experiments/migrations/0039_add_users_to_default_team.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2 on 2023-10-12 12:55
+from django.conf import settings
+from django.db import migrations
+
+
+def add_users_to_default_team(apps, schema_editor):
+    Team = apps.get_model("teams", "Team")
+    User = apps.get_model(settings.AUTH_USER_MODEL)
+    Membership = apps.get_model("teams", "Membership")
+    default_team = Team.objects.order("id").first()
+    for user in User.objects.all():
```

Comment:
I don't totally understand this logic / why this is necessary. If Users were part of a different team would it not make sense to explicitly have them added to the team manually, as opposed to doing it automatically?

Reply:
Prior to this PR all the data was visible in every team. After this PR it will only be visible in the 'default team' so all existing users who are not already part of that team will lose access. Inviting them could be done manually but this has the same effect.